### DISCUSSION TITLE
【灌水】救命！我迷失在零号空洞了

### DISCUSSION THREAD

**Discussion by Coolliangyuu** [share121/inter-knot#3153] [其他]
体力不够用怎么办

**Comment by RaphaelHider** [comment]
可以去前文明遗迹供奉巴巴托斯或者摩拉克斯的神像获得祝福提高体力

**Reply by Coolliangyuu** [reply]
我超 猿

**Comment by zzztjs** [comment]
再等几天所有神像都可以给体力了，不用去找他俩的了

**Comment by yutong903** [comment]
喝咖啡<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >
